FunctionsSdkE2ETests/nuget.config

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
-    <add key="localSource" value="..\artifacts" />
     <add key="nuget.org" value="https://www.nuget.org/api/v2/" />    
```

Comment:
Add a `<clear />` element also - it will be expected by supply chain analysis

Reply:
correct, done.